docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,111 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+Validation and Verification Criteria
+------------------------------------
 
-   all of this
+Obejective
+++++++++++
+External
+ - Model results should be checked against local TB epidemiology
```

Comment:
I would take out "TB" here so that it can be applied as a guide for all cause models

Reply:
Oh, sorry - I was wrong about this!! I though that this was a PR for the template, not for LTBI. This looks good in that case :)